Homework13/PageObjects/WebTablePage.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.Collections.ObjectModel;
+using System.Text;
+using Homework13.Common.WebElements;
+using OpenQA.Selenium;
+
+namespace Homework13.PageObjects
+{
+    public class WebTablePage
+    {
+        private MyWebElement _addNewEntryButton = new(By.XPath("//button[@id='addNewRecordButton']"));
+        private MyWebElement _submitButton = new(By.XPath("//button[@id='submit']"));
+        private MyWebElement _form = new(By.XPath("//form"));
+        private MyWebElement _firstName1 = new(By.XPath("((//div[@role='rowgroup'])[1]//descendant::div[@role='gridcell'])[1]"));
+        private MyWebElement _row = new(By.XPath("//div[@role='rowgroup']"));
+        private MyWebElement _searchInput = new(By.XPath("//input[@id='searchBox']"));
+        private MyWebElement _firstNameInput = new(By.XPath("//input[@id='firstName']"));
+        private MyWebElement _lastNameInput = new(By.XPath("//input[@id='lastName']"));
+        private MyWebElement _emailInput = new(By.XPath("//input[@id='userEmail']"));
+        private MyWebElement _ageInput = new(By.XPath("//input[@id='age']"));
+        private MyWebElement _salaryInput = new(By.XPath("//input[@id='salary']"));
+        private MyWebElement _departmentInput = new(By.XPath("//input[@id='department']"));
+        private MyWebElement _editButton = new(By.XPath("//span[@title='Edit']"));
+        private MyWebElement _deleteButton = new(By.XPath("//span[@id='delete-record-1']"));
+        private MyWebElement _closeButton = new(By.XPath("//span[contains(text(),'×')]"));
+
+        public void ClickAddNewEntryButton() => _addNewEntryButton.Click();
+
+        public void ClickCloseButton() => _closeButton.Click();
+
+        public void ClickSubmitButton() => _submitButton.Click();
+
+        public void ClickEditEntryButton() => _editButton.Click();
+
+        public void ClickDeleteEntryButton() => _deleteButton.Click();
+
+        public string GetFormClass() => _form.GetAttribute("class");
+
+        public string GetFirstName1Value() => _firstName1.Text;
+
+        public ReadOnlyCollection<IWebElement> GetRows()
```

Comment:
Please refactor - it's better to make method which return IWebElement private. So in tests you don't use IWebElement, just interact with page methods